### PR TITLE
test updates due to new cosmos version

### PR DESCRIPTION
**Pull request by tamarrow**

### CODE REVIEW

**Comment by tamarrow** [comment]
#431

File: cli/tests/integrations/test_package.py [hunk 1]
```diff
@@ -952,22 +951,45 @@ def _install_chronos(
...
+        watch_all_deployments()
+
+
 def _list(args=['--json'],
           stdout=b'[]\n'):
     assert_command(['dcos', 'package', 'list'] + args,
                    stdout=stdout)
 
 
-def _list_remove_nulls(args=['--json'], stdout=b'[]\n'):
```
Comment (BenWhitehead) [inline cli/tests/integrations/test_package.py]: Yay!

File: cli/tests/integrations/test_package.py [hunk 2]
```diff
@@ -1022,16 +1044,3 @@ def _repo_add(args=[], repo_list=[]):
 def _repo_remove(args=[], repo_list=[]):
     assert_command(['dcos', 'package', 'repo', 'remove'] + args)
     assert_command(['dcos', 'package', 'repo', 'list'], stdout=repo_list)
-
-
-# issue 431
-def _remove_nulls(output):
```
Comment (BenWhitehead) [inline cli/tests/integrations/test_package.py]: Yay!

**Comment by BenWhitehead** [comment]
Seems okay to me once builds are passing.

**Pull request merged**